cpp/src/io/parquet/page_enc.cu

```diff
@@ -115,13 +121,42 @@ __global__ void __launch_bounds__(512) gpuInitPageFragments(PageFragment *frag,
   __syncthreads();
   start_row = blockIdx.y * fragment_size;
   if (!t) {
-    s->col.num_rows            = min(s->col.num_rows, max_num_rows);
+    s->col.num_rows = min(s->col.num_rows, max_num_rows);
+    // fragment size = 5000. max_num_rows = total num rows in column. So frag num rows = 5000 except
```

Comment:
This comment is unlikely to follow any updates to fragment_size/MAX_FRAGMENT_SIZE, maybe use the name(s) instead of the current value.